src/pagefault.c

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Do we really need a new file only for the page fault handler? Seems like it should live in the pageable module. All other modules that implement interrupts follow that pattern